warehouse/ingest-core/src/main/java/datawave/ingest/mapreduce/job/MultiRFileOutputFormatter.java

```diff
@@ -145,7 +145,7 @@ public static void setCompressionTableDisallowList(Configuration conf, Set<Strin
     protected static Set<String> getCompressionTableDisallowList(Configuration conf) {
         String tableListString = conf.get(COMPRESSION_DISALLOWLIST);
         if (tableListString == null) {
-            return Collections.EMPTY_SET;
+            return new HashSet<>();
```

Comment:
Wouldn't `Collections.emptySet()` resolve the warning and still preserve immutability?

Reply:
Yep.  Changing all 3, resolving as I do it.